admin_manual/installation/apps_supported.rst

```diff
@@ -38,6 +38,7 @@ Below is the list of apps supported for Nextcloud 12. Supported here means that
 * Provisioning API
 * Server info (monitoring app)
 * Sharebymail
+* Socialsharing-plugins (Diaspora, email, Facebook, Google+ and Twitter)
```

Comment:
we should not name them plugins - just social sharing (because the title is "Supported **apps**") 😉